README.rst

```diff
@@ -40,6 +40,11 @@ Creating a Managed Connection
     >>> manager
     <ManagedConnection: postgres:///postgres (closed)>
 
+It is possible to pass keywor arguments to the underlying psycopg2.connect() function like this:
```

Comment:
There's a spelling error here (keywor) and "psycopg2.connect()" should be probably enclosed in double backticks (see the reference to `psycopg2.connection` in the first paragraph.)

Also, the syntax isn't highlighted here (needs `.. code::` block declaration) and lines should be prefixed with three angle brackets (`>>>`) for doctest compatibility.